### PR TITLE
SCMOD-4279: Update the openSUSE base image to Leap 15.0

### PR DESCRIPTION
**Pull request by michael-bryson**
JIRA: https://jira.autonomy.com/browse/SCMOD-4279

### CODE REVIEW

File: src/main/docker/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-FROM opensuse:latest
+FROM opensuse/leap:latest
```
Comment (michael-bryson) [inline src/main/docker/Dockerfile:17]: Should we use the 15.0 or 15 tag instead of latest?

Comment (dermot-hardy) [reply src/main/docker/Dockerfile:17]: No latest is fine - the first thing we do is update the OS packages anyway.

File: pom.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
     <groupId>com.github.cafapi</groupId>
     <artifactId>opensuse-base-image</artifactId>
-    <version>1.2.0-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
```
Comment (dermot-hardy) [inline pom.xml]: As discussed, there doesn't seem to be a breaking change here (so far as we can see).

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Comment by michael-bryson** [comment]
Review comments addressed.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**